doc/api/assert.md

```diff
@@ -778,6 +778,23 @@ let err;
 //     at errorFrame
 ```
 
+## `assert.initMustSettle()`
+
+Initialises async hook for `mustSettle()` function.
```

Comment:
This should include an explanation about *why* this separate function is needed (specifically, that the hook needs to be added and enabled before any tracked promises are created).

Alternatively, using a method that is more like `common.mustCall()` and `common.mustNotCall()` would avoid the need to use an async hook at all.

Reply:
Our [style guide] standardizes on US spellings, so:

```suggestion
Initializes async hook for `mustSettle()` function.
```

Also, should `async hook` be a specific object/class, like `AsyncHook` or something?